_data/projects/html-duration-picker.yml

```
@@ -0,0 +1,12 @@
+name: HTML-DURATION-PICKER.JS
```

This doesn't need to be all-caps, and can just be how the project is named in the README:

```suggestion
name: html-duration-picker.js
```

---

@shiftkey thank you for the suggestion. You are right. I've applied the changes. Thanks again